packages/ui/src/components/Button/Button.css.ts

```diff
@@ -0,0 +1,227 @@
+import { style, styleVariants } from '@vanilla-extract/css'
+import { minWidth, theme } from '../../theme'
+
+export const baseButton = style({
+  // opt out of double tap to zoom to immediately respond to taps
+  touchAction: 'manipulation',
+  borderRadius: theme.radius.sm,
+  whiteSpace: 'nowrap',
+  lineHeight: 1,
+  transition: `background-color ${theme.transitions.hover}, box-shadow 0.15s cubic-bezier(0.47, 0.03, 0.49, 1.38) 0s`,
+  display: 'inline-flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+
+  cursor: 'pointer',
+  ':disabled': {
+    cursor: 'default',
+  },
+
+  ':focus-visible': {
+    boxShadow: theme.shadow.focus,
+  },
+})
+
+export const fullWidthStyles = style({
+  width: '100%',
+})
+
+const shadow = style({
+  boxShadow: theme.shadow.default,
+  backdropFilter: 'blur(30px)',
+})
+
+export const buttonVariant = styleVariants({
+  primary: [
+    baseButton,
+    {
+      backgroundColor: theme.colors.gray1000,
+      color: theme.colors.textNegative,
+
+      '@media (hover: hover)': {
+        '&:hover': {
+          backgroundColor: theme.colors.grayTranslucent900,
+        },
+      },
+
+      ':active': {
+        backgroundColor: theme.colors.grayTranslucent900,
+      },
+
+      ':not([data-loading])': {
+        ':disabled': {
+          backgroundColor: theme.colors.gray200,
+          color: theme.colors.textDisabled,
+        },
```

Comment:
I think those are not being applied.  You might want to change it to the following:

```suggestion
      selectors: {
        ':not([data-loading]):disabled': {
           backgroundColor: theme.colors.gray200,
           color: theme.colors.textDisabled,
        },
      }
```

Reply:
Good catch @Guilherme :raised_hands: